pkg/network/ovn_kubernetes.go

```diff
@@ -340,6 +340,8 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		data.Data["OVN_MULTI_NETWORK_POLICY_ENABLE"] = true
 	}
 
+	data.Data["OVN_PERSISTENT_IPS_ENABLE"] = true
```

Comment:
I think we should default this to false, thus leave nothing installed for 4.16. 

During the course of 4.17 we can then introduce the feature gate / enable the installation.

Reply:
But why? It only makes sense if it is tech preview. That is the determining factor.

Reply:
If we don't want to support this in any way in 4.16, we don't even need this PR at all.

Reply:
Yeah, we're past the KubeVirt 4.16 feature freeze, which means this can only be ready for 4.17. 

We can ignore this PR for now.

Reply:
OK, I'm trying to get the API PR going forward, so we can proceed w/ this one.